pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -634,6 +686,10 @@ func (p *criStatsProvider) getContainerUsageNanoCores(stats *runtimeapi.Containe
 		return nil
 	}
 
+	if stats.Cpu != nil && stats.Cpu.UsageNanoCores != nil {
```

Comment:
add comment to explain why there's two code paths here and why in this case we avoid using cached `UsageNanoCores`